recipes/libavif/bld.bat

```diff
@@ -0,0 +1,25 @@
+mkdir build
+cd build
+if errorlevel 1 exit /b 1
+
+:: Other codecs cannot be enabled because they are not on conda-forge
+cmake .. -GNinja ^
+-DCMAKE_INSTALL_PREFIX="%PREFIX%" ^
```

Comment:
This needs to be `%LIBRARY_PREFIX%` on Windows, as things are installed into `%PREFIX%\Library\{include, bin, lib...}`

Reply:
I think you can use `%CMAKE_ARGS%`

Reply:
I don't think CMAKE_ARGS exists for windows (at least it isn't in the log.) in either case, CMAKE_ARGS only defines the compilers to use, not the prefix.

Reply:
either way, we need %LIBRARY_PREFIX% :)

Reply:
```suggestion
-DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
```